packages/test/end-to-end-tests/src/test/compat.spec.ts

```diff
@@ -54,17 +53,14 @@ describe("loader/runtime compatibility", () => {
             container.on("closed", (error) => containerError = containerError || error !== undefined);
 
             dataObject = await requestFluidObject<TestDataObject>(container as IFluidRouter, "default");
-
```

Comment:
nit: opProcessingController changes seems unrelated / can be staged as separate RR

Reply:
Yeah. The problem was discover thru the opProcessingController changes.  So I want to make sure that the solution works for it.